docs/ref/models/database-functions.txt

```diff
@@ -1114,6 +1114,15 @@ It can also be registered as a transform. For example::
     >>> # Get vectors whose radians are less than 1
     >>> vectors = Vector.objects.filter(x__radians__lt=1, y__radians__lt=1)
 
+``Random``
+----------
+
+.. class:: Random(**extra)
+
+.. versionadded:: 3.2
+
+Returns a random value in the range ``[0.0, 1.0)``.
```

Comment:
I wonder if everyone reading this will immediately know the [) syntax - I have to look up which is which every time. It might be better to state which of the bounds is inclusive and which is exclusive.

Reply:
I guess that's fair. That syntax is also used for bounds by the range type in PostgreSQL, but we've hidden that away in Django behind `django.contrib.postgres.fields.range.RangeBoundary`.

I'll change this to ``0.0 ≤ x < 1.0`` and that is what PostgreSQL, MySQL, and MariaDB use in their documentation.